salt/_modules/metalk8s_os.py

```diff
@@ -0,0 +1,59 @@
+"""
+MetalK8s OS module
+"""
+
+__virtualname__ = "metalk8s_os"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get_kubereserved() -> dict:
```

Comment:
nit: Let's not type here we do not use type anywhere else in salt modules

```suggestion
def get_kubereserved():
```